snooty.toml

```diff
@@ -26,11 +26,11 @@ language = "C#"
 mongo-community = "MongoDB Community Edition"
 mongo-enterprise = "MongoDB Enterprise Edition"
 docs-branch = "master"                                                                                          # always set this to the docs branch (i.e. master, 1.7, 1.8, etc.)
-version-number = "2.22"                                                                                         # always set this to the driver branch (i.e. 1.7.0, 1.8.0, etc.)
+version-number = "2.23"                                                                                         # always set this to the driver branch (i.e. 1.7.0, 1.8.0, etc.)
 version = "v{+version-number+}"
 example = "https://raw.githubusercontent.com/mongodb/docs-csharp/{+docs-branch+}/source/includes/code-examples"
 stable-api = "Stable API"
-api-root = "https://mongodb.github.io/mongo-csharp-driver/{+version-number+}/html"
+api-root = "https://mongodb.github.io/mongo-csharp-driver/2.23.0/api/"
```

Comment:
FYI: My PR will overwrite this change

Reply:
Ok, reverting. I will handle any merge conflicts.